### PR TITLE
msmpi: FORTRAN90's .mod files, test suite

### PR DESCRIPTION
**Pull request by okhlybov**
- Install precompiled .mod files for FORTRAN90's USE statement.
- Add TclTest-based test suite to check for C, C++ and FORTRAN basic MPI build & run capability.

### CODE REVIEW

File: mingw-w64-msmpi/PKGBUILD [hunk 1]
```diff
@@ -4,13 +4,14 @@ _realname=msmpi
...
 pkgver=10.1.1
-pkgrel=3
+pkgrel=4
 pkgdesc="Microsoft MPI SDK (mingw-w64)"
 arch=('any')
 url="https://docs.microsoft.com/en-us/message-passing-interface/microsoft-mpi"
 license=('MIT')
 makedepends=("${MINGW_PACKAGE_PREFIX}-binutils")
 depends=("${MINGW_PACKAGE_PREFIX}-gcc" "${MINGW_PACKAGE_PREFIX}-gcc-fortran")
+optdepends=("${MINGW_PACKAGE_PREFIX}-tcl: build & run test suite")
```
Comment (lazka) [inline mingw-w64-msmpi/PKGBUILD:14]: should this be checkdepends then?

File: mingw-w64-msmpi/PKGBUILD [hunk 2]
```diff
@@ -21,6 +22,13 @@ source=(
   'mpifptr.h.i686'
   'msmpi.def.x86_64'
   'msmpi.def.i686'
+  'hello_mpi.c'
```
Comment (lazka) [inline mingw-w64-msmpi/PKGBUILD:25]: Where are these files coming from?

Comment (okhlybov) [reply mingw-w64-msmpi/PKGBUILD:25]: These are publically available MPI examples.

File: mingw-w64-msmpi/PKGBUILD (hunk 1 above)
Comment (okhlybov) [reply mingw-w64-msmpi/PKGBUILD:14]: Probably not. This test will fail during CI as it requires MS-MPI runtime to be installed.

**Pull request merged**